test/integration/spec/connect.js

```diff
@@ -47,6 +47,12 @@ const { trackPriority: { PRIORITY_HIGH, PRIORITY_LOW, PRIORITY_STANDARD } } = re
 
 const safariVersion = isSafari && Number(navigator.userAgent.match(/Version\/([0-9.]+)/)[1]);
 
+// placeholder ensures all variations of TEST_STABILITY end up with some tests to run.
+describe('placeholder', () => {
```

Comment:
I am not sure if this is needed? What happens if we do not have a placeholder?

Reply:
when mocha executes a file and did not find any tests to run, it fails the run. so w/o such placeholder a run with TEST_STABILITY=stable would fail if all tests in a file are marked unstable.

Reply:
Thanks for the suggestion of `failOnEmptyTestSuite=false`. Now we can get rid of this and other placeholder tests.